onnxruntime/core/session/inference_session.h

```diff
@@ -132,6 +139,8 @@ class InferenceSession {
   */
   common::Status LoadCustomOps(const std::vector<std::string>& dso_list);
 
+  common::Status AddCustomOpDomains(const std::vector<OrtCustomOpDomain*>& ops);
```

Comment:
const OrtCustomOpDomain*?

Reply:
I'm not sure if const is appropriate here, since creating kernels might modify the custom op classes?